WordPress/src/main/res/values/strings.xml

```diff
@@ -2271,6 +2272,21 @@
     <string name="role_follower">Follower</string>
     <string name="role_viewer">Viewer</string>
 
+    <string name="invite_links_section_title">Invite Link</string>
+    <string name="invite_links_get_status_retry">Get links retry</string>
```

Comment:
Maybe we could change this one to something a bit more user-friendly, eg. "Refresh link status"?